docs/source/development/widget.md

```diff
@@ -337,9 +337,18 @@ To configure a new Volto widget follow these steps.
 
 1.  Finally, restart Volto by pressing {kbd}`Ctrl-C` inside the active frontend shell session, then restart it with the following command.
 
-    ```shell
-    make frontend-start
-    ```
+    ````{tab-set}
+      ```{tab-item} Frontend add-on Cookieplone project
+        ```shell
+        make start
+        ```
+      ```
+      ```{tab-item} Fullstack Cookieplone project
+        ```shell
+        make frontend-start
+        ```
+      ```
+    ````
```

Comment:
That's creative syntax, and I'm surprised it works. Let's stick with supported MyST syntax, and nest by increasing the count of backticks, in case it becomes strict.

```suggestion
    `````{tab-set}
    ````{tab-item} Frontend add-on Cookieplone project
    ```shell
    make start
    ```
    ````
    ````{tab-item} Fullstack Cookieplone project
    ```shell
    make frontend-start
    ```
    ````
    `````
```